bacon.toml

```diff
@@ -27,6 +27,7 @@ command = [
 	"--all-targets",
 ]
 need_stdout = false
+beep_on_end = true
```

Comment:
I feel like defaulting to false might be better. I don't know if everyone will want this enabled.

Reply:
It defaults to false. This specific line in bacon's bacon.toml is only for easier tests of this PR

Reply:
Oh ok makes sense.